index.bs

```diff
@@ -1563,6 +1563,37 @@ Stats {#streaming-stats}
 TODO
 
 
+Protocol Extensions {#protocol-extensions}
+===================
+
+Open Screen agents may exchange extension messages that are not defined by this
+specification.  This could be used for experimentation, customization or other
+purposes.
+
+If an agent wishes to support extension messages, the author of the extension
```

Comment:
Nit: awkward sentence, since the subject of the first clause (agent) is not linked to the subject of the second clause (the extension).

Maybe something like:

To add new extension messages, authors must register a capability ID with a range of message type keys in a public registry (location to be determined). Agents may then query that public registry, and indicate that they accept an extension message type by including the corresponding capability ID in the
`capabilities` field of its `agent-info` message.

Reply:
Accepted suggestion with a few tweaks; agents aren't required to query the registry itself to support an extension, as it's likely the capabilities will be hard coded or set by configuration flags.